.github/workflows/lfric_test.yml

```diff
@@ -75,6 +75,58 @@ jobs:
         pip install .[test]
         pip install jinja2
 
+    # PSyclone, compile and run MetOffice gungho_model on GPU
+    - name: LFRic GungHo with OpenACC offload
+      run: |
+        # Set up environment
+        source /apps/spack/psyclone-spack/spack-repo/share/spack/setup-env.sh
+        spack load lfric-build-environment%nvhpc
+        source .runner_venv/bin/activate
+        export PSYCLONE_LFRIC_DIR=${GITHUB_WORKSPACE}/examples/lfric/scripts
+        export PSYCLONE_CONFIG_FILE=${PSYCLONE_LFRIC_DIR}/KGOs/lfric_psyclone.cfg
+        # The LFRic source must be patched to workaround bugs in the NVIDIA
+        # compiler's namelist handling.
+        rm -rf ${HOME}/LFRic/gpu_build
+        mkdir -p ${HOME}/LFRic/gpu_build
+        cp -r ${HOME}/LFRic/lfric_apps_${LFRIC_APPS_REV} ${HOME}/LFRic/gpu_build/lfric_apps
+        cp -r ${HOME}/LFRic/lfric_core_50869 ${HOME}/LFRic/gpu_build/lfric
+        cd ${HOME}/LFRic/gpu_build
+        patch -p1 < ../lfric_${LFRIC_APPS_REV}_nvidia.patch
+        # Update the compiler definitions to build for GPU
+        cp ../nvfortran.mk lfric/infrastructure/build/fortran/.
+        cp ../nvc++.mk lfric/infrastructure/build/cxx/.
+        # Update the PSyclone commands to ensure transformed kernels are written
+        # to working directory.
+        cp ../psyclone.mk lfric/infrastructure/build/psyclone/.
```

Comment:
Could you put `../lfric_${LFRIC_APPS_REV}_nvidia.patch` `../nvfortran.mk` `../nvc++.mk` and `../psyclone.mk` in somewhere in the examples/lfric/scripts?

I addition to being able to capture the whole test, I would also like to have all the files to be able to replicate it in my workspace easily.

Reply:
Yes, good idea. I also realised that the transformation script adds OpenMP if it fails to add OpenACC but we weren't building with OMP enabled. Hopefully this will still work...